src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppEvent.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.sendletter.logging;
+
+public final class AppEvent {
+
+    /**
+     * Used to track event prior converting letter to servicebus message to track all income.
```

Comment:
income?

Reply:
perhaps i meant traffic 😂  or smth like that